app/main.py

```diff
@@ -1,2 +1,46 @@
+from typing import Any
+
+
 class Dictionary:
-    pass
+    def __init__(self) -> None:
+        self.size = 0
+        self.capacity = 8
+        self.threshold = int(self.capacity * 2 / 3)
+        self.hash_table = [[] for _ in range(self.capacity)]
+
+    def __setitem__(self, key: int, value: Any) -> None:
+        if self.size == self.threshold:
+            old_hash_table = self.hash_table
```

Comment:
it would be better to make another method for resizing

Reply:
Ok. I created 'resize' method